integration-tests/search.test.ts

```diff
@@ -536,220 +539,148 @@ describe('searches with patient permission levels', () => {
         };
         encounterResource = (await adminClient.post('Encounter', encounterResourceJson)).data;
         await waitForResourceToBeSearchable(adminClient, encounterResource);
+
+        // create patient with linked org
+        const orgJson = {
+            resourceType: 'Organization',
+            name: 'Health Level Seven International',
+            alias: ['HL7 International'],
+        };
+        orgResource = (await adminClient.post('Organization', orgJson)).data;
+        const randPatient = randomPatient();
+        randPatient.managingOrganization.reference = `Organization/${orgResource.id}`;
+        testPatient = (await adminClient.post('Patient', randPatient)).data;
+        // wait for the patient to be asynchronously written to ES
+        await waitForResourceToBeSearchable(adminClient, testPatient);
+        sherlockHolmes = (await adminClient.get(`Patient/${idsOfFhirResources.sherlockHolmes}`)).data;
     });
 
     afterAll(async () => {
         await adminClient.delete(`Encounter/${encounterResource.id}`);
+        await adminClient.delete(`Organization/${orgResource.id}`);
+        await adminClient.delete(`Patient/${testPatient.id}`);
     });
 
-    test('tests with only patient.read scope', async () => {
-        const client = await getFhirClient('fhirUser patient/Patient.read launch/patient profile openid', false);
-        const sherlockHolmes = (await adminClient.get(`Patient/${idsOfFhirResources.sherlockHolmes}`)).data;
-
-        const getPatient = await client.get('Patient');
-        expect(getPatient.data.total).toBe(2); // Should return Mycroft Holmes as a seealso
-        expect(getPatient.data.entry[1].resource).toMatchObject(sherlockHolmes);
-
-        await expectResourceToNotBePartOfSearchResults(
-            client,
-            { url: 'Patient', params: { _include: 'Patient:organization' } },
-            {
-                resourceType: 'Organization',
-            },
-        );
-
-        await expectResourceToNotBePartOfSearchResults(
-            client,
-            { url: 'Patient', params: { _revinclude: 'Encounter:subject' } },
-            encounterResource,
-        );
+    // Since the tests for patient scope and user (as Patient) scope are the same results,
+    // we can group them
+    const scopes: string[] = ['patient', 'user'];
+    describe.each(scopes)('searches with patient/user (as Patient) scopes', (scope) => {
```

Comment:
```suggestion
    describe.each(scopes)('searches with %p (as Patient) scopes', (scope) => {
```

This should name the test correctly